drivers/tmp006/include/tmp006.h

```diff
@@ -1,5 +1,6 @@
 /*
  * Copyright (C) 2014 PHYTEC Messtechnik GmbH
+ *               2017 HAW Hamburg
```

Comment:
I see the point but it is inconsistent with the other driver implementations. I would prefer to keep it in `drivers/include`.